TriblerGUI/test/widgets/network_explorer/js/test_utilities.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * This file tests the utilities.js file with unit tests
+ */
+
+assert = require("assert");
+utilities = require("TriblerGUI/widgets/network_explorer/js/utilities.js");
+config = require("TriblerGUI/widgets/network_explorer/js/style_config.js");
+
+describe("utilities.js", function () {
+    describe('formatBytes', function () {
+        it('should return correct formats', function () {
+            var data = [
+                [0, "0.000 B"],
+                [999, "999.0 B"],
+                [1000, "1.000 kB"],
+                [1001, "1.001 kB"],
+                [9999, "9.999 kB"],
+                [20000, "20.00 kB"],
+                [200000, "200.0 kB"],
+                [1000000, "1.000 MB"],
+                [1001000, "1.001 MB"],
+                [9999000, "9.999 MB"],
+                [1000000000, "1.000 GB"],
+                [1001000000, "1.001 GB"],
+            ];
+
+            data.forEach(function (set) {
+                assert.equal(utilities.formatBytes(set[0]), set[1]);
+
+                // Check negatives, except zero
+                if(set[0] > 0) assert.equal(utilities.formatBytes(-set[0]), "-" + set[1]);
```

Comment:
Will do, thanks! We don't want another [`goto fail` issue](https://www.dwheeler.com/essays/apple-goto-fail.html)...